### ISSUE TITLE
Consider usage of a standard ORM

### ISSUE THREAD

**Issue by trickeydan** [srobo/nemesis#11]
`nemesis.sqlitewrapper.KeyedSqliteThing` looks like a basic implementation of an ORM.

We could reduce the amount of code we need to maintain by replacing this with an ORM library, such as SQLAlchemy.

**Comment by PeterJCLaw** [comment]
Yes, definitely. At the moment `nemesis` has very minimal requirements, in part because it dates from a time before virtualenvs being popular (and possibly even possible, I can't recall when they first appeared) and getting dependencies installed on the deployment server was a bit complicated because (I think) this also pre-dates us having the config puppetised.

**Comment by PeterJCLaw** [comment]
As an aside: I've actually been considering rebuilding nemesis based on Django as that would get us a lot of stuff for free.
